reviews/views.py

```diff
@@ -1,3 +1,52 @@
-from django.shortcuts import render
+from datetime import datetime
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Avg
+
+from rooms.models import Room
+
+class ReviewView(View):   
+    def get(self, request, room_id):
+        try:
+            limit   = int(request.GET.get('limit', 6))
+            offset  = int(request.GET.get('offset', 0))*6           
```

Comment:
offset에 *6이 있는 이유가 무엇일까요?

Reply:
프론트에서 '페이지'를 offset으로 받아서 페이지네이션을 처리하려고 했습니다.

Reply:
6을 그대로 하드코딩을 해버리면 기획이 바꼈을 때 코드를 수정해야 됩니다.
페이지를 받는다고 하면 아래와 같이 수정해야 할 것 같습니다.
```suggestion
            page  = int(request.GET.get('page', 1))
            offset = (page-1) * limit           
```

Reply:
넵 말씀해주신대로 수정하겠습니다!